src/parser/shaman/enhancement/modules/checklist/Component.tsx

```diff
@@ -51,7 +51,7 @@ const EnhancementShamanChecklist = ({ castEfficiency, combatant, thresholds }: a
 
       <Rule
         name="Maintain your buffs"
-        description={"You should maintain your buffs in order to passivly increase your damage done to targets without refreshing them to early."}
+        description="You should maintain your buffs in order to passivly increase your damage done to targets without refreshing them to early."
```

Comment:
I know this may not have been you, but I think it's important to fix the rule description:

"~~to~~ **too** early"

Also, the wording here feels a bit off overall. Maybe something like this?

"Maintaining your buffs passively increases your damage output. However, refreshing them too early leads to time wasted that could be spent on direct damage spells."